provision/testdata/with_overrides.inp.yaml

```diff
@@ -99,6 +99,7 @@ logging:
   controller_log_level: debug
   hostagent_log_level: debug
   opflexagent_log_level: info
+  operator_log_level: debug
```

Comment:
info or debug ?

Reply:
We set non-default value in overrides input yaml, so debug value is correct for operator_log_level.

Reply:
noted